app/services/availability_service.py

```
@@ -428,20 +431,29 @@ async def check_availability(self, request: AvailabilityRequest) -> Availability
+                        if isinstance(room_detail.priceConfig, dict):
+                            total_price = self.calculate_total_price(
+                                room_detail=room_detail,
+                                date_str=request.date,
+                                hour_slots=hour_slots,
+                                available_slots=res.available_slots if is_partial else None,
+                            )
+                        else:
+                            # NOTE: list-backed config 방의 가격은 _apply_policies에서 이미 산정됨.
+                            #       여기까지 도달했다면 _apply_policies 계산이 실패한 것이므로 None을 그대로 보존함.   
+                            total_price = res.estimated_price
+                    except Exception as e:
```

Ruff (0.15.4)

[warning] Do not catch blind exception: `Exception`

(BLE001)